### DISCUSSION TITLE
How to Add Tabs Between Items When Copying and Pasting <td> Tags in a Table?

### DISCUSSION THREAD

**Discussion by guystrike** [6pac/SlickGrid#1090] [Q&A]
Hi,
I am using SlickGrid to create a table in my application. When users copy and paste data from the table, all the items are currently pasted as a single line of text.

I would like to modify this behavior so that items in the table are separated by tabs (like in a spreadsheet) when copied.

Is there a built-in option in SlickGrid to enable this, or is there a recommended approach to achieve it?

Thank you in advance for your assistance!

**Comment by ghiscoding** [comment]
See this [Example](https://6pac.github.io/SlickGrid/examples/example-excel-compatible-spreadsheet-dataview.html) then select & copy+paste any section from 1 grid to the other (or to Excel). I think that demo plugin does copy Tabs
